flex-checks/src/main/java/org/sonar/flex/checks/LineLengthCheck.java

```diff
@@ -57,24 +54,21 @@
   public int maximumLineLength = DEFAULT_MAXIMUM_LINE_LENHGTH;
 
   @Override
-  public void setCharset(Charset charset) {
-    this.charset = charset;
+  public List<AstNodeType> subscribedTo() {
+    return Collections.emptyList();
   }
 
   @Override
   public void visitFile(@Nullable AstNode astNode) {
-    List<String> lines = Collections.emptyList();
+    String fileContent = getContext().fileContent();
+    String[] lines = fileContent.split("\\r?\\n");;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty statement. ([squid:EmptyStatementUsageCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.flex%3Aflex&branch=SONARFLEX-105&issues=AWGPc9z6Ol9lQhPgXcMu&open=AWGPc9z6Ol9lQhPgXcMu)

[ISSUE_KEY]: AWGPc9z6Ol9lQhPgXcMu